doc/source/development/contributing_environment.rst

```diff
@@ -26,14 +26,27 @@ with a full pandas development environment.
 
 **Docker Commands**
 
-Pass your GitHub username in the ``DockerFile`` to use your own fork::
+Build the Docker image::
 
     # Build the image pandas-yourname-env
     docker build --tag pandas-yourname-env .
-    # Run a container and bind your local forked repo, pandas-yourname, to the container
-    docker run -it --rm -v path-to-pandas-yourname:/home/pandas-yourname pandas-yourname-env
+    # Or build the image by passing your GitHub username to use your own fork
+    docker build --build-arg gh_username=yourname --tag pandas-yourname-env .
 
-Even easier, you can integrate Docker with the following IDEs:
+Run Container::
+
+    # Run a container and bind your local repo to the container
+    docker run -it -w /home/pandas --rm -v path-to-local-pandas-repo:/home/pandas pandas-yourname-env
+
+.. note::
+    If you bind your local repo for the first time, you have to build the C extensions afterwards.
+    Run the following command inside the container::
+
+        python setup.py build_ext -j 4
+
+    You might need to rebuild the C extensions if/when you merge your repo with upstream/main.
```

Comment:
```suggestion
    You need to rebuild the C extensions anytime the Cython code in `pandas/core/_libs` changes. This most frequently occurs when changing or merging branches. 
```

Reply:
Even better would be to separate out this into a new section, something like "Rebuilding the C extensions" and going into a bit more detail as to the why/how/when. But certainly okay to leave it here for this PR.

Reply:
1) Is `pandas/core/_libs` right? Because such a folder does not exist. I think you mean `pandas/_libs`?

> Even better would be to separate out this into a new section, something like "Rebuilding the C extensions"... But certainly okay to leave it here for this PR.

2) That's a good idea. I can create a new issue if that is OK for you.

Reply:
Yes, an issue would be great.

Reply:
>Yes, an issue would be great.

I created one.